### DISCUSSION TITLE
FreeRDP Parameter with Multimonitor

### DISCUSSION THREAD

**Discussion by rabiie3** [FreeRDP/FreeRDP#7666] [Q&A]
Hey there,
i am new to github and having some trouble with the parameter **/multimon:force**.

i am using freerdp on TinyCore http://tinycorelinux.net/
i have already asked in the community from TinyCore but they are saying that freeRDP might has the problem not the OS:
http://forum.tinycorelinux.net/index.php/topic,25575.0.html#lastPost

The Problem i am having is, that when i run freerdp like this:
`xfreerdp /sec:rdp /network:auto /multimon:force +multitransport /f /scale:140  /d:domain /v:server:3389`
there are some flages that won't work with Multimonitors

But when i use the **/scale** parameter on one Monitor it get to work like this:
`xfreerdp /sec:rdp /network:auto +multitransport /f /scale:140  /d:domain /v:server:3389`

another problem is the **Full window drag**
the freerdp Manual says that Full window drag is disabled by default but again that work just with one monitor
`+window-drag (default:off)   Full window drag`

Xfreerdp version is 2.5.0
`xfreerdp /version
This is FreeRDP version 2.5.0 (2.5.0)`

i don't know if that is a bug 😅

is there anyone who can help ?

thank you

**Comment by akallabeth** [comment]
`/multimon` currently does not support setting an arbitrary scale (that is limited to a single monitor/windowed mode)

**Reply by rabiie3** [reply]
Hi akallabeth,

what about the full drag window ? when pressing **alt** and dragging, i can drag the whole session. 
shouldn't be that disabled by default ? like i said it is disabled with one monitor but not with` /multimon`

>  /multimon currently does not support setting an arbitrary scale (that is limited to a single monitor/windowed mode)

is there any other way i can scale the session a little bit more ?

**Reply by akallabeth** [reply]
* `window-drag` is just a flag (that might be ignored by the server)
* What do you mean with pressing `alt` ? Behavior depends on if you have `keyboard-grab` enabled or not.
* Currently only way to scale is in single monitor/window mode

**Reply by rabiie3** [reply]
this is the exact command line i am using:
`xfreerdp /sec:rdp /network:auto /drive:USB-Stick,/mydisk /f /multimon:force +multitransport +auto-reconnect -themes -wallpaper /cert-ignore /bpp:15 /kbd:0x00000407 /d:"domain" /v:"server":3389`

> What do you mean with pressing alt ?

i mean when i press the key **alt** and drag any window i see, then i can drag the whole RDP Session and see the background from my linux system. What i don't want to happen. can i disable that ? 

> Behavior depends on if you have keyboard-grab enabled or not.

i think it is by default enabeld

**Reply by rabiie3** [reply]
i mean when you hold the key **alt** and drag any window you see. (only with /multimon:force)
i think that this is a bug...😅
